app/Models/UserSettings.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use App\Models\User;
+
+class UserSettings extends Model
+{
+    use HasFactory;
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'guided_use_toggle',
+        'audio_level',
+        'user_id',
+    ];
+
+    /**
+     * Get the user who owns these settings.
+     * 
+     * @var array
+     */
+
+    public function user()
+    {
+        return $this->belongsTo(User::class);
```

Comment:
I think on the [User](https://github.com/peeley/lucy/blob/master/app/Models/User.php) class you can also add the converse now, like

```php
public function settings()
{
    return $this->hasOne(UserSettings::class);
}
```

if you want to add that (or not, this isn't used yet anyway).

looks good though, merge away!

Reply:
I did add that function to the User class, can you see it on your end?

Reply:
oh my b, I have no idea how I missed that. nvm!